data/grab_data.rb

```diff
@@ -0,0 +1,49 @@
+require 'nokogiri'
+require 'open-uri'
+require 'pry'
+
+doc = Nokogiri::HTML(open("http://en.wikipedia.org/wiki/List_of_Chicago_%22L%22_stations"))
+
+table =  doc.css("table").select{|a| a.to_s.include?("Rapid transit stations on the Chicago")}.first
```

Comment:
Line is too long. [100/80]<br>Unnecessary spacing detected.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.